app/org/sagebionetworks/bridge/models/DateRangeResourceList.java

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.bridge.models;
+
+import java.util.List;
+
+import org.joda.time.LocalDate;
+
+public class DateRangeResourceList<T> {
```

Comment:
Added serialization test. Not deserialization because we never deserialize it on the server, so didn't want to add @JsonCreator et al when we don't do it, to support the test.